tests/functional/m/method_hidden.py

```diff
@@ -106,13 +109,24 @@ def default(self, o):
 class Parent:
     def __init__(self):
         self._protected = None
+        self._protected_two = None
 
 
 class Child(Parent):
     def _protected(self):  # [method-hidden]
         pass
 
 
+class CachedChild(Parent):
+    @ft.cached_property
+    def _protected(self):
+        pass
+
+    @functools.cached_property
+    def _protected_two(self):
```

Comment:
There is no actual definition `cached_property`, so `safe_infer` returns `UnInferable`, and this line should have been pointed out by the pylint.

By adding the actual definition, I think this line will be detected, any good ideas?

https://github.com/pylint-dev/pylint/blob/1ef3b0be3c018ab4f2928932ab1f46ee244b79bb/pylint/checkers/classes/class_checker.py#L1261-L1262

Reply:
It's the expected behavior 👍 We should not raise when something is uninferable (unless the real functools.cached_property is always uninferable).